### ISSUE TITLE
"This extension may soon no longer be supported"

### ISSUE THREAD

**Issue by cglong** [Rob--W/stackexchange-notifications#33]
Chrome is reporting that Desktop Notifications for Stack Exchange may soon be disabled. Based on timing of this, I'm assuming the extension is written using Manifest v2, which would put the deadline at June 2025.

**Comment by Rob--W** [comment]
That's right. This is one of the extensions that I still need to migrate to MV3.
